calcom/cal.com
# fix: use callback url in signup

## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/4383

https://user-images.githubusercontent.com/53316345/207410279-a33d80da-c54a-4e7d-9765-56e2e4cc273c.mov

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

1) Create an account in /signup 
2) then try logging in you should be redirected to /login
